### ISSUE TITLE
Moove the createEvent function from ERC20 contract to 

### ISSUE THREAD

**Issue by Trottoir** [massalabs/massa-as-sdk#68]

**Comment by gregLibert** [comment]
@Trottoir can you clarify this issue ?

**Comment by Thykof** [comment]
the task is to move this https://github.com/massalabs/massa-sc-contracts/blob/main/assembly/utils/event.ts into massa-as-sdk

**Comment by gregLibert** [comment]
> the task is to move this https://github.com/massalabs/massa-sc-contracts/blob/main/assembly/utils/event.ts into massa-as-sdk

Is it really needed ?

**Comment by Thykof** [comment]
yes because this function is used in multiple standard contracts, se we need to avoid duplicating this.

**Comment by Thykof** [comment]
blocked by https://github.com/massalabs/massa-sc-contracts/issues/25
